### PR TITLE
Updating the downloads page with 7.2.6 version

### PR DESCRIPTION
**Pull request by bbarani**
### Description
Updating the downloads page with 7.2.6 version artifact links

 
### Issues Resolved
NA

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: content/download/releases/v7-2-6.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "7.2.6"
+date: 2024-08-01
```
Comment (roshkhatri) [inline content/download/releases/v7-2-6.md:3]: 7.2.6 was released on 2024-07-31
```suggestion
date: 2024-07-31
```

Comment (bbarani) [reply content/download/releases/v7-2-6.md]: Fixed it now.

File: content/download/releases/v7-2-6.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    container_registry:
+        - 
+            name: "Docker Hub"
+            link: https://hub.docker.com/r/valkey/valkey/
+            id: "valkey/valkey"
+            tags:
+                - "7.2.6"
+                - "7.2.6-bookworm"
+                - "7.2.6-alpine"
+                - "7.2.6-alpine3.19"
```
Comment (roshkhatri) [inline content/download/releases/v7-2-6.md:17]:
```suggestion
                - "7.2.6-alpine3.20"
```

Comment (bbarani) [reply content/download/releases/v7-2-6.md]: Updated it now.

**Review by madolson — APPROVED** [review]

**Pull request merged**